shared/transforms/relabelids.py

```diff
@@ -116,37 +128,48 @@ def ensure_by_xccdf_referenced_oval_def_is_defined_in_oval_file(xccdftree, ovalt
     # Fixes: https://github.com/OpenSCAP/scap-security-guide/issues/1095
     # Fixes: https://github.com/OpenSCAP/scap-security-guide/issues/1098
 
-
-    xccdfrules = xccdftree.findall(".//{%s}Rule" % xccdf_ns)
-    for rule in xccdfrules:
+    for rule in xccdftree.findall(".//{%s}Rule" % xccdf_ns):
         xccdfid = rule.get("id")
-        if xccdfid is not None:
-            # Search OVAL ID in OVAL document
-            ovalid = ovaltree.find(".//{%s}definition[@id=\"%s\"]" % (oval_ns, xccdfid))
-            if ovalid is None:
-                # Search same ID in XCCDF document
-                check = rule.find(".//{%s}check[@system=\"%s\"]" % (xccdf_ns, oval_ns))
-                # Skip XCCDF rules not referencing OVAL checks
-                if check is not None:
-                    checkcontentref = check.find(".//{%s}check-content-ref" % xccdf_ns)
-                    if checkcontentref is not None:
-                        try:
-                            checkcontentref_hrefattr = checkcontentref.get('href')
-                        except KeyError:
-                            # @href attribute of <check-content-ref> is required by XCCDF standard
-                            print("\nError: Invalid OVAL <check-content-ref> detected! Exiting..")
-                            sys.exit(1)
+        if xccdfid is None:
+            # TODO: We probably should issue a warning here or even error..
```

Comment:
Yes, that is the case before and after this PR, I just noticed it so I added a comment